docs/src/main/paradox/google-cloud-pub-sub-grpc.md

```diff
@@ -29,6 +29,40 @@ The table below shows direct dependencies of this module and the second tab show
 
 @@dependencies { projectId="google-cloud-pub-sub-grpc" }
 
+## Build setup
+
+The Alpakka Google Cloud Pub/Sub gRPC library contains the classes generated from [Google's protobuf specification](https://github.com/googleapis/java-pubsub/tree/master/proto-google-cloud-pubsub-v1/).
+
+To call the API via gRPC, the ALPN Java agent is required to be set up correctly.
+
+### Maven
+
+Configure your project to use the Java agent for ALPN and add `-javaagent:...` to your startup scripts as described in the @extref:[Akka gRPC documentation](akka-grpc:/buildtools/maven.html#starting-your-akka-grpc-server-from-maven).
```

Comment:
I *think* this is only needed on JDK's 8 and earlier - we should double-check and update the Akka gRPC docs to make this clear as well.

Reply:
That's right, I've heard that before... please let me know when you confirmed it.

Reply:
I just confirmed it with the Java akka-grpc-quickstart, added https://github.com/akka/akka-grpc/issues/772